style.css

```diff
@@ -0,0 +1,21 @@
+body {
+    background-color: blueviolet;
+}
+
+
```

Comment:
Please kindly remove one space from here! Remember is crucial to comply what styling guidelines.